velox/functions/sparksql/Arithmetic.h

```diff
@@ -35,6 +35,28 @@ struct RemainderFunction {
     if (UNLIKELY(n == 0)) {
       return false;
     }
+    if constexpr (std::is_floating_point_v<TInput>) {
+      handle_floating_point(result, a, n);
+    } else {
+      handle_integral(result, a, n);
+    }
+    return true;
+  }
+
+ private:
+  template <typename TInput>
+  void handle_floating_point(TInput& result, const TInput& a, const TInput& n) {
```

Comment:
Would you follow the naming conventions to use camelCase for functions? Ditto for the other functions.
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#naming-conventions